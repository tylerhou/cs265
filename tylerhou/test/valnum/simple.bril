@main {
  a: int = const 1;
  b: int = const 2;
  c: int = const 3;
  sum1: int = add a b;
  sum2: int = add a b;
  prod: int = mul sum1 sum2;
  ret prod;
}
