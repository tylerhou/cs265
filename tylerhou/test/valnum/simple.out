@main {
.__b0:
  a: int = const 1;
  b: int = const 2;
  c: int = const 3;
  sum1: int = const 3;
  sum2: int = const 3;
  prod: int = const 9;
  ret prod;
}
